SkietbaanBE/SkietbaanBE/Models/UserGroup.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SkietbaanBE.Models {
+    public class UserGroup {
+        public int Id { get; set; }
+
+        public int UId { get; set; }
+        [ForeignKey("UId")]
+        public User User { get; set; }
```

Comment:
Why are we coding these they should happen by Magic
If we name our colums corectly EF should create the foreign keys itself , have a look and let me know